commands/info.md

```diff
@@ -305,6 +305,10 @@ Here is the meaning of all fields in the **stats** section:
 *   `total_writes_processed`: Total number of write events processed
 *   `io_threaded_reads_processed`: Number of read events processed by the main and I/O threads
 *   `io_threaded_writes_processed`: Number of write events processed by the main and I/O threads
+*   `acl_access_denied_auth`: Number of AUTH authentication failures
```

Comment:
Since we're agnostic to the command invoked (`AUTH` or `HELLO`):

```suggestion
*   `acl_access_denied_auth`: Number of  authentication failures
```